compiler/src/typed/env.re

```diff
@@ -306,6 +307,17 @@ module TycompTbl = {
       keys2,
     );
   };
+
+  let rec get_all = tbl =>
```

Comment:
A bit of a hack but I wasn't sure how else to implement https://github.com/grain-lang/grain/pull/1586#discussion_r1071645473